src/content-script.js

```diff
@@ -100,7 +100,8 @@ function autoChangeUsername() {
 			process.env.NODE_ENV === 'development' && console.log('[autoChangeUsername] got valid username')
 			
 			const usernameAlreadyStored = localStorage.getItem('owncast_username')
-			if (usernameAlreadyStored) {
+			const usernameCustom = localStorage.getItem('owncast_custom_username_set')
+			if (usernameAlreadyStored && usernameCustom) {
 				process.env.NODE_ENV === 'development' && console.log('[autoChangeUsername] found username', usernameAlreadyStored, 'associated with this instance - not changing username automatically')
```

Comment:
I think the naming of the variables is confusing now - also, this solution won't work on old versions.

Maybe:
```javascript
const customUsernameSet = localStorage.getItem('owncast_custom_username_set')
const potentialStoredUsername = localStorage.getItem('owncast_username')

if (customUsernameSet || potentialStoredUsername) {
    ...
```

Reply:
The varbiable owncast_username is always set in version 0.0.8 with a auto generated username. A logical OR is not the solution :(

Reply:
A logical AND would break compatibility with old instances (I guess). However, if we release the new version of the addon after the release of 0.0.8, it should be fine with a AND.

How do you think about renaming the variables, though?